linux/bash_utils.bsh

```diff
@@ -72,4 +74,39 @@ function print_bash_stack()
       printf "%${stack_depth_pad}s. %-${funcname_pad}s:%-${lineno_pad}s %s\n" "$((i-${1-0}))" "${FUNCNAME[i]:+${FUNCNAME[i]}()}" "${BASH_LINENO[i-1]}" "${BASH_SOURCE[i]}" >&2
     done
   fi
-}
\ No newline at end of file
+}
+
+#**
+# .. function:: run_bash_rc_file
+#
+# Depending on how bash is compiled (``-DSYS_BASHRC``), spawning bash with a custom RC files is not always straight forward. This function will normalize the behavior to that no matter what version/configuration of ``bash`` you are running, your RC files will be loaded before running the desired bash command.
```

Comment:
```suggestion
# Depending on how bash is compiled (``-DSYS_BASHRC``), spawning bash with a custom RC file is not always straight forward. This function will normalize the behavior so that no matter what version/configuration of ``bash`` you are running, your RC files will be loaded before running the desired bash command.
```